tests/Doctrine/Tests/ORM/Tools/SchemaValidatorTest.php

```diff
@@ -675,3 +700,31 @@ abstract class Issue9095AbstractChild extends Issue9095Parent
 class Issue9095Child extends Issue9095AbstractChild
 {
 }
+
+/** @Entity */
+class InvalidEntity3
+{
+    /**
+     * @var int
+     * @Id
+     * @Column
+     */
+    protected $key;
+
+    /**
+     * @Column(type="boolean")
+     */
+    protected int $property;
+}
+
+/** @Entity */
+class InvalidChildEntity extends InvalidEntity3
+{
+    /** @Column(type="string") */
+    protected int $property;
```

Comment:
Note: `doctrine/orm` does not (AFAIK) allow parent/child properties with the same name. The suggestion I sent you before to add an example with inheritance was mostly about having parent class properties that are **not** visible to the child class